topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaContractTest.java

```diff
@@ -85,36 +85,37 @@
 
 /**
  * An external requirement of the service is to connect to a pair of kafka topics.
- * The upstream "masked-resource" topic is written to by the attribute-masking-service and read by this service.
- * The downstream "masked-resource-offset" topic is written to by this service and read by the filtered-resource-service.
+ * The upstream "attribute-masking-service" topic is and read by this service.
+ * The downstream "filtered-resource-offset" topic is written to by this service.
```

Comment:
The names included in the old diff are the names of the topics, not the names of the services, revert those changes

Reply:
Ok